public/configs/env.json

```diff
@@ -42,12 +42,14 @@
             "chain": {
                "name": "DYDX",
                "denom": "adv4tnt",
+               "exponents": 18,
```

Comment:
nit: should we use `decimals`?
seems to be the convention, e.g our [chain info](https://v4-teacher.vercel.app/testnets/testnet_info#chain-info)